diff --git a/src/lib/logging.ts b/src/lib/logging.ts
index 72c20d7..cca9bfb 100644
--- a/src/lib/logging.ts
+++ b/src/lib/logging.ts
@@ -96,7 +96,7 @@ class WebSocketLogTransport extends Transport {
           client.sendEvent({
             source: "driver",
             event: "logging",
-            formattedMessage: info[this.messageSymbol as any],
+            formattedMessage: info[this.messageSymbol as any] as string,
             ...info,
           }),
         );
